CIP-0021/README.md

```diff
@@ -84,7 +86,9 @@ If a transaction contains a pool registration certificate, then it must not cont
 
 - any other certificate;
 - any withdrawal;
-- mint entry.
+- mint entry;
+- any collateral input;
+- any required signer.
```

Comment:
I'm not sure if `script_data_hash` should also be included here? Or is it just items which require witnesses?

Reply:
Must be included here, too. Ledger code:
```
security_policy_t policyForSignTxScriptDataHash(const sign_tx_signingmode_t txSigningMode)
{
	switch (txSigningMode) {
	case SIGN_TX_SIGNINGMODE_ORDINARY_TX:
	case SIGN_TX_SIGNINGMODE_MULTISIG_TX:
	case SIGN_TX_SIGNINGMODE_PLUTUS_TX:
		SHOW();
		break;

	case SIGN_TX_SIGNINGMODE_POOL_REGISTRATION_OWNER:
	case SIGN_TX_SIGNINGMODE_POOL_REGISTRATION_OPERATOR:
		DENY();
		break;

	default:
		ASSERT(false);
	}
```